### PR TITLE
Create fr.lua

### PR DESCRIPTION
**Pull request by TonybynMp4**

### CODE REVIEW

File: locales/fr.lua [hunk 1]
```diff
@@ -0,0 +1,36 @@
+local Translations = {
+    error = {
+        no_deposit = "$%{value} dépôt requis",
+        cancelled = "Annulé",
+        vehicle_not_correct = "Ce n'est pas un véhicule Commercial!",
```
Comment (restray) [inline locales/fr.lua:5]:
```suggestion
        vehicle_not_correct = "Ce n'est pas un véhicule commercial!",
```

**Review by restray — CHANGES_REQUESTED** [review]
The translation is almost perfect, I've add some suggest you may (or not) apply!

(PS: Merci beaucoup pour les traductions 🥳)

**Comment by TonybynMp4** [comment]
> The translation is almost perfect, I've add some suggest you may (or not) apply!
> 
> (PS: Merci beaucoup pour les traductions 🥳)

Non-Official French translator for your service .😂

**Review by restray — APPROVED** [review]
Everything seems fine to me! 👍🏻

**Pull request merged**